.github/workflows/doc.nuxeo.com.yml

```
@@ -0,0 +1,66 @@
+name: Build and Deploy doc.nuxeo.com
+
+on:
+  push:
+    branches:
+      - master
+      - development
+
+jobs:
+  documentation_ci:
+    name: Build & Publish documentation to ${{ github.ref_name == 'master' && 'production' || github.ref_name }}
```

```suggestion
    name: Build & Publish documentation to ${{ github.ref_name == 'master' && 'production' || 'development' }}
```

it's better if you fallback explicitly to that env, otherwise github will not fail and automatically create a new empty env if doesn't already exists

---

I was not aware of that behavior... then go for that solution